DESCRIPTION

```diff
@@ -23,13 +23,13 @@ Authors@R: c(
     person("GlaxoSmithKline LLC", role = c("cph", "fnd")),
     person("Amgen Inc.", role = c("cph", "fnd"))
     )
-Description: A toolbox for programming oncology specific Clinical Data Standards Interchange Consortium
-    (CDISC) compliant Analysis Data Model (ADaM) datasets in R. ADaM datasets are a
+Description: Programming oncology specific Clinical Data Standards Interchange Consortium
```

Comment:
just noted we got the ordering wrong - it should be: Clinical Data Interchange Standards Consortium

Reply:
Good catch! I have fixed it.

We have the same mistake in admiral.

Reply:
I made an issue for admiral